syntax = "proto3";

package proto.live;

import "RegionSimpleInfo_v2.proto";

message QueryRegionListHttpRsp_v2 {
  int32 retcode = 1;
  repeated RegionSimpleInfo_v2 region_list = 2;
  bytes client_secret_key = 5;
  bytes client_custom_config_encrypted = 6;
  bool enable_login_pc = 7;
}
